consensus/src/experimental/buffer_manager.rs

```diff
@@ -38,52 +39,12 @@ pub struct SyncRequest {
 }
 
 pub struct OrderedBlocks {
-    pub blocks: Vec<ExecutedBlock>,
+    pub ordered_blocks: Vec<ExecutedBlock>,
```

Comment:
this ExecutedBlock is a little confusing, we should strip it into raw Block somewhere

Reply:
I feel this is a bit wasting resources as we are converting it back and forth only for readability.

Reply:
this can happen in the OrderingStateComputer, but maybe later